README.md

```diff
@@ -499,6 +500,9 @@ public static IActionResult Run(
     return new CreatedResult($"/api/addproduct", product);
 }
 ```
+## Known Issues
+
+- Attempting to use output bindings against tables with columns of data types `NTEXT`, `TEXT`, or `IMAGE` is not supported, as these types are deprecated and do not work with the `OPENJSON` function. Upserting data will fail in this case.
```

Comment:
Output bindings against tables with columns of data types `NTEXT`, `TEXT`, or `IMAGE` are not supported and data upserts will fail. These types [will be removed](https://docs.microsoft.com/sql/t-sql/data-types/ntext-text-and-image-transact-sql) in a future version of SQL Server and are not compatible with the `OPENJSON` function used by this Azure Functions binding.

Reply:
This is great. Thank you!